templates/include/glean/glean.1.schema.json

```diff
@@ -26,7 +26,7 @@
     },
     "dot_separated_short_id": {
       "type": "string",
-      "pattern": "^[a-z_][a-z0-9_]{0,29}\\.[a-z_][a-z0-9_]{0,29}$"
+      "pattern": "^[a-z_][a-z0-9_]{0,29}(\\.[a-z_][a-z0-9_]{0,29})+$"
```

Comment:
There is now no length limit here, is that expected?

Reply:
Good point.  I think we need to figure out what this limit should be, because obviously it should be longer than before...

@georgf ?